go-controller/pkg/ovn/base_network_controller_secondary.go

```diff
@@ -307,6 +307,14 @@ func (bsnc *BaseSecondaryNetworkController) addLogicalPortToNetworkForNAD(pod *k
 		if err != nil {
 			return err
 		}
+
+		if kubevirt.IsPodOwnedByVirtualMachine(pod) && kubevirt.IsAllowedForMigration(bsnc.NetInfo) {
+			ops, err = bsnc.updateLogicalSwitchPortsForLiveMigrationPods(pod, lsp, ops, nadName, switchName)
+			if err != nil {
+				return err
+			}
+		}
```

Comment:
We can move this away from the if statement `if requiresLogicalPort {`

Let's just move to https://github.com/ovn-kubernetes/ovn-kubernetes/pull/4774/commits/999b8fa890f8c59a43e71db1644c690bcd200526#diff-7ade52078b2eac50eeb5f96fb67df39d804cf8b48592206691f748334f785b58R366